src/SentryOne.UnitTestGenerator.Core/Helpers/Generate.cs

```diff
@@ -40,6 +41,41 @@ public static MethodDeclarationSyntax Method(string name, bool isAsync, bool isS
             return method;
         }
 
+        public static string CleanName(string sourceName)
+        {
+            if (string.IsNullOrWhiteSpace(sourceName))
+            {
+                return sourceName;
+            }
+
+            var first = true;
+            var builder = new StringBuilder();
+            foreach (var ch in sourceName)
```

Comment:
Can this be handled through the Roslyn Syntax libraries? I know you can test for a valid identifier, but I'm unsure if it will let you replace.

Reply:
Yeah I didn't find anything to clean a name, just to tell you if it was valid or not 👍